data/json/monsters/zed-winged.json

```
@@ -142,5 +143,49 @@
       "SWARMS",
       "FLIES"
     ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_spawn_raptor_shady",
+    "name": { "str_sp": "shady flesh-raptor" },
```

```suggestion
    "name": { "str": "shadow raptor" },
```
Let's make 'em sound cooler